electron-app/src/services/rotkehlchen-api.ts

```diff
@@ -230,12 +273,20 @@ export class RotkehlchenApi {
     });
   }
 
+  private validate_status_put_settings(status: number) {
```

Comment:
nitpick: Since in TypeScript the functions are usually in camelcase can we make sure that we follow this for function naming? (I will probably enforce it by eslint in the future, and we can add exceptions in a few models that are coming from JSON where this makes sense) :)